### ISSUE TITLE
Sidebar: Contacts management

### ISSUE THREAD

**Issue by valeriansaliou** [prose-im/prose-app-macos#43]

**Comment by nesium** [comment]
How are we going to handle presence subscriptions?

Since we're working in a team/company environment it would be cumbersome if you'd have to send subscription requests to each of your teammates.

But that's a deviation from the standard XMPP and could only be handled on the server since presence subscriptions are one-way (e.g. both parties have to allow subscription).

**Comment by valeriansaliou** [comment]
You're right, Prose will come w/ a Prosody server already configured to auto-manage the roster + subscriptions for all @domain that are the same as yours. So in this regard, we will only have to deal w/ incoming 'subscribe' presence types, which if the user is using a Prose-backed XMPP server configuration would only come from external domains, _if the server administrator enabled server to server federation, that is if they opted for the server not to be isolated for security reasons (eg. prevent phishing and scams)_.

The contact accept/reject interface was designed in this view (in the "invites" tab):

![Image](https://user-images.githubusercontent.com/1451907/174758035-8fc3771e-9bfd-4a5c-b39a-a760cdcce49d.jpg)
